tests/CMakeLists.txt

```diff
@@ -113,6 +113,16 @@ if(AVIF_ENABLE_GTEST)
     target_include_directories(avifopaquetest PRIVATE ${GTEST_INCLUDE_DIRS})
     add_test(NAME avifopaquetest COMMAND avifopaquetest)
 
+    add_executable(avifpng16btest gtest/avifpng16btest.cc)
+    target_link_libraries(avifpng16btest aviftest_helpers ${GTEST_LIBRARIES})
+    target_include_directories(avifpng16btest PRIVATE ${GTEST_INCLUDE_DIRS})
```

Comment:
Nit: I suggest changing "16b" to "16bit" or "16bpc" in the test/file names and the test image name.